### PR TITLE
Putting together Fault Proof Upgrade Runbook

### PR DESCRIPTION
**Pull request by sbvegan**
**Description**

The goal of this PR is to put together a reusable run book we can follow to generate upgrade tasks for chains who have completed the MCP L1 upgrade and now need to get to the fault proof contract set.

This is a draft for now, we'll serve as a space for me to hammer out the details. However, we'll follow the general run book model we've used in the past of:

1. preparing the task for a network
2. simulating, validating, and executing that task
3. then using that task as a template for future upgrades

### CODE REVIEW

File: runbooks/fault-proofs.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
+# Key Handover Runbook
+
+This document describes how to generate upgrade playbooks to upgrade chains from `op-contracts/v1.3.0` (MCP) to `op-contracts/v1.X.0` (Fault Proofs).
```
Comment (maurelian) [inline runbooks/fault-proofs.md:3]: Note: determine if the target is v1.5.0 or v1.6.0. 
(I assume that's why it's given as `v1.x.0`)

File: runbooks/fault-proofs.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+
+## Upgrades Involved
+
+This runbook will apply the following upgrade:
+
+- [Fault Proofs](#fault-proofs-upgrade-7)
+
+Each upgrade is described below.
+The Extended Pause upgrade is only applied to chains currently on the Bedrock commit, corresponding to the [`op-contracts/v1.0.0`](https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts%2Fv1.0.0) tag in the [Optimism monorepo](https://github.com/ethereum-optimism/optimism).
```
Comment (maurelian) [inline runbooks/fault-proofs.md:12]: Extended pause is 1.2.0: https://github.com/ethereum-optimism/optimism/releases/tag/op-contracts/v1.2.0

**Comment by maurelian** [comment]
@sbvegan is this still wanted?

**Comment by sbvegan** [comment]
@maurelian we can close this one for now and add a new one later. We're shifting our approach on this so I'll make sure thats buttoned up before I put in a PR

**Pull request closed without merging**